doc/user/posix.md

```diff
@@ -204,7 +204,7 @@ library works in conjunction with DFuse and allows the interception of POSIX I/O
 calls and issue the I/O operations directly from the application context through
 libdaos without any application changes.  This provides kernel-bypass for I/O data
 leading to improved performance.
-To use this set the LD_PRELOAD to point to the shared library in the DOAS install
+To use this set the LD_PRELOAD to point to the shared library in the DAOS install
```

Comment:
How does it know how to spell _DAOS_?

Reply:
It doesn't, but I do.

Reply:
But if it doesn't know how to spell it, how does it know it's not a spelling error?

Reply:
Ah I see.  codespell checks for common typos, not against a dictionary.  The real advantage it has is that it can automatically correct things a lot of the time.

Reply:
Ahhh.  I see.  Interesting.